### PR TITLE
Use separate `_view` and `_download` links for action file attachments

### PR DESCRIPTION
**Pull request by nmajor25**
Shortcut Story ID: [sc-32343]

Use the `file.meta._view` URL for the filename link. And use the `file.meta._download` URL for the download button link.

Also, update the API request parameters to `?urls=download,view`.

### CODE REVIEW

File: test/integration/patients/sidebar/action-sidebar.js [hunk 1]
```diff
@@ -802,7 +803,8 @@ context('action sidebar', function() {
               created_at: '2019-08-25T14:15:22Z',
             },
             meta: {
-              download: 'https://www.bucket_name.s3.amazonaws.com/patients/1/HRA%20v2.pdf',
+              view: 'https://www.bucket_name.s3.amazonaws.com/patients/1/view/HRA%20v2.pdf',
+              download: 'https://www.bucket_name.s3.amazonaws.com/patients/1/download/HRA%20v2.pdf',
```
Comment (nmajor25) [inline test/integration/patients/sidebar/action-sidebar.js:807]: Don't know what the difference between these two links will be in the prod environment.

But this just creates two different links and tests that they are used in the correct spots. Filename vs. Download button.

**Review by paulfalgout — APPROVED** [review]

File: test/integration/patients/sidebar/action-sidebar.js (hunk 1 above)
Comment (shadowhand) [reply test/integration/patients/sidebar/action-sidebar.js:807]: Technically the path won't be different, but rather the query string will be modified. NBD.

**Review by shadowhand — APPROVED** [review]

File: test/integration/patients/sidebar/action-sidebar.js (hunk 1 above)
Comment (nmajor25) [reply test/integration/patients/sidebar/action-sidebar.js:807]: Ah gotcha. We can update this test later to be more accurate compared to qa/prod.

Comment (paulfalgout) [reply test/integration/patients/sidebar/action-sidebar.js:807]: meh it's fine, these strings could be `view-url-goes-here` and it's enough for these integration tests.

Comment (shadowhand) [reply test/integration/patients/sidebar/action-sidebar.js:807]: Yup, exactly. The important part is just differentiating between download/upload/view.

Comment (nmajor25) [reply test/integration/patients/sidebar/action-sidebar.js:807]: Works for me!

**Pull request merged**